lms/templates/learner_dashboard/_dashboard_navigation_programs.html

```diff
@@ -0,0 +1 @@
+## This file is left intentionally blank and can be overridden using theming
```

Comment:
Need to add `<%page expression_filter="h"/>`. This should fix the quality errors.